openmc/universe.py

```diff
@@ -871,6 +877,12 @@ def auto_mat_ids(self, val):
         cv.check_type('DAGMC automatic material ids', val, bool)
         self._auto_mat_ids = val
 
+    @property
+    def material_assignment(self):
```

Comment:
These values aren't going to exclusively be material assignments. The `NAME` tag in the `.h5m` file also contains boundary conditions, temperatures, etc.

https://pshriwise.github.io/dagmc-docs/filespec/index.html

Reply:
Though it looks like this could be removed in favor of the `material_names` property below?